pkg/lite-apiserver/cert/cert_manager.go

```diff
@@ -56,10 +56,13 @@ func (m *CertManager) Init() error {
 	return nil
 }
 
-func (m *CertManager) Load(name string) *http.Transport {
+func (m *CertManager) GetTransport(name string) *http.Transport {
+	if len(name) == 0 {
+		return m.defaultTr
+	}
 	t, e := m.transportMap[name]
```

Comment:
s/e/ok

Reply:
```go
if trans, ok := m.transportMap[name]; ok {
   return trans
}
return m.defaultTr

```

Reply:
fixd in #44 as that pr refactor transport and cert